### DISCUSSION TITLE
how show grid line like navisworks or revit

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#5434] [Q&A]
now i open the imodel,and show grid ,but not show like in navisworks .
note: in naviswoks ,the green line is the grid line .

![grid2](https://user-images.githubusercontent.com/18022720/233757862-7862ef51-b7c7-4f52-aa66-d19bb304462c.png)
![grid](https://user-images.githubusercontent.com/18022720/233757866-1e5fab09-bd54-4c69-aaf0-8df948e7bcea.png)

**Comment by arnab-git-hub** [comment]
Hi @paprophet, In the first image, the imodel grid (view) is not part of the model and is not an element. It is a visual measurement and alignment aid, and can be turned on/off for any view.

In the second image, the green lines are actually part of the building grid layout, that is generally a series of straight vertical and/or horizontal lines. These are part of the model and are needed to be drawn. 

Can you please share the Navisworks file with me via email at arnab.gogoi [at] bentley [dot] com to take a look?

**Reply by arnab-git-hub** [reply]
Hi @aruniverse, it seems the itwin viewer does not show building grid layouts. Can you please confirm?
